packages/data/domain/validation.schemas.ts

```diff
@@ -20,10 +20,30 @@ export type NewApplicationModel = z.infer<typeof newApplicationSchema>;
 export const interviewSelectSchema = createSelectSchema(interviews);
 export type InterviewModel = z.infer<typeof interviewSelectSchema>;
 
+export const interviewWithCommentSchema = interviewSelectSchema.extend({comments: z.array(z.object({id: z.string(), interview_id: z.string(), comment_date: z.number(), comment: z.string(), pinned: z.boolean()}))})
```

Comment:
@souravnub maybe, having a comments schema (similar to `interviewSelectSchema` and the `InterviewModel`) in this file would be a bit easier. This way, you could reference it here, instead of embedding it. What do you think?